doc/TESTING.md

```diff
@@ -80,3 +66,22 @@ Portable packages are also available. They can be run from your home directory,
 Unstable macOS installers are generated after every commit to the master branch, and should only be used for testing.
 
 - [Download Installer](https://nightly.link/nicotine-plus/nicotine-plus/workflows/packaging/master/macos-installer.zip)
+
+## Git
```

Comment:
If you move this section out, pip should also be moved.

I would revert this change, since setting up an environment on Windows/macOS is rather annoying/time-consuming. The Windows/macOS-specific builds are better options.

Reply:
> If you move this section out, pip should also be moved.

@redactedscribe seems to have a better approach in #2037